src/test/java/org/dz/solution/task33/Task33Test.java

```diff
@@ -0,0 +1,66 @@
+package org.dz.solution.task33;
+
+import org.junit.Test;
+
+import java.io.*;
+import java.util.*;
+
+
+public class Task33Test {
+    private static final String SOURCE_DATA_FILE = "src/main/resources/org/dz/solution/task33/MainFile";
+    private static final String RESULT_FILE = "src/main/resources/org/dz/solution/task33/OutputFile";
+    private static final String[] SYMBOLS = new String[]{",", "-", "\\.", ";", "(", ")", ":", "\"", "«", "»", "„", "“"};
+    public static final String WORD_DELIMITER = " ";
+
+    @Test
+    public void test() {
+    }
+
+    private void calculateSkippedNumbers() {
+        List<String> textFromFile = ReadingAFile(SOURCE_DATA_FILE);
+        wordCount(textFromFile);
+        writeWordCountToFile(RESULT_FILE);
+    }
+
+    private List<String> ReadingAFile(String sourceDataFile) {
+        List<String> array = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(sourceDataFile))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                for (String symbol : SYMBOLS) {
+                    line = line.replaceAll(symbol, WORD_DELIMITER);
+                }
+                String[] lineWords = line.split(WORD_DELIMITER);
+                Collections.addAll(array, lineWords);
+            }
+        } catch (IOException e) {
+            System.out.println("File reading failed");
+            e.printStackTrace();
+        }
+        return array;
+    }
+
+    private void wordCount(List<String> array) {
+        Map<String, Integer> wordCount = new HashMap<>();
+        for (String word : array) {
+            if (wordCount.containsKey(word)) {
+                int count = wordCount.get(word);
+                wordCount.put(word, count + 1);
+            } else {
+                wordCount.put(word, 1);
+            }
+        }
+    }
+
+    public static void writeWordCountToFile(String resultFile) {
+        Map<String, Integer> wordCount = new HashMap<>();
+        try (PrintWriter writer = new PrintWriter(new FileWriter(resultFile))){
+            for (Map.Entry<String, Integer> entry : wordCount.entrySet()) {
+                writer.println(entry.getKey() + " " + entry.getValue());
+            }
+        }catch (IOException e){
+            System.out.println("Data not written to file");
```

Comment:
"Data not written to file" is good but it would be better like this "Writing to the file failed"